src/ctapipe/image/muon/__init__.py

```diff
@@ -4,13 +4,18 @@
     ring_completeness,
     ring_containment,
 )
-from .fitting import kundu_chaudhuri_circle_fit
+from .fitting import kundu_chaudhuri_circle_fit, taubin_circle_fit
 from .intensity_fitter import MuonIntensityFitter  # noqa: F401
 from .processor import MuonProcessor  # noqa: F401
-from .ring_fitter import MuonRingFitter  # noqa: F401
+from .ring_fitter import (
+    MuonRingFitter,  # noqa: F401
+    kundu_chaudhuri_taubin,
+)
 
 __all__ = [
     "kundu_chaudhuri_circle_fit",
+    "taubin_circle_fit",
```

Comment:
The first two entries here are actual fit methods, while the third one is a thin wrapper. The other two also have thin wrappers that are not exposed here. @maxnoe was it a deliberate omission, or they also have to be added?

Reply:
Not sure if this is an omission or to fix a "multiple references" issue with sphinx.

If the `MuonRingFitter` is also exposed in `ctapipe.image`, then the latter. A class can only be documented at two places not at three in sphinx.

Reply:
Edit: indeed, the high-level API classes are exposed in `ctapipe.image`, this is why they are not added to `__all__` here.

Reply:
this is true, however the enum of fit methods is not documented and one will never know from the doc about the possibility to have the third, chained fitting method... So it should be fixed somehow.

Reply:
The function should be added here, yes

Reply:
And we should make sure that the available methods are in the documentation, but I think that can be addressed in a separate PR:

https://ctapipe.readthedocs.io/en/latest/api/ctapipe.image.MuonRingFitter.html#ctapipe.image.MuonRingFitter